packages/schema/docs/build/schema.md

```diff
@@ -2080,14 +2080,24 @@ Key | Required | Type | Description
 --- | -------- | ---- | -----------
 `window` | **yes** | `integer` | The timeframe, in seconds, within which the system tracks the number of invocations for an action. The number of invocations begins at zero at the start of each window.
 `limit` | **yes** | `integer` | The maximum number of invocations for an action, allowed within the timeframe window.
-`scope` | no | `array`[`string` in (`'user'`, `'auth'`, `'account'`)] | The granularity to throttle by. You can set the scope to one or more of the following: 'user' - Throttles based on user ids.  'auth' - Throttles based on auth ids. 'account' - Throttles based on account ids for all users under a single account. By default, throttling is scoped to the account.
+`key` | no | `string` | The key to throttle with in combination with the scope. This should be unique to the operation. While actions of different integrations with the same key and scope will never share the same limit, actions of the same integration with the same key and scope will do when "action" is not in the scope. User data provided for the input fields can be used in the key with the use of the curly braces referencing. For example, to access the user data provided for the input field "test_field", use `{{bundle.inputData.test_field}}`. Note that a required input field should be referenced to get user data always.
```

Comment:
Is it worth mentioning what happens if the key is not provided?

Reply:
Might be helpful, though implied with "in combination with the scope"

Reply:
That may be helpful, although implied with "in combination with the scope".

Perhaps even more helpful would be to explain `key` and `scope` together in the description of the schema instead of here individually in the props.

We could explain that throttling will be shared by all actions with the same `key` and that the values referenced by `scope` are mixed into it, and that `scope` defaults to `['account']`.

Pseudo code may also help here:

```ts
const usedKey = (scope || ['account'])
  .map(s => values[s])
  .concat(key, app)
  .join('-');
```

Same likely applies to `LockObjectSchema`.

Reply:
Updated the descriptions by moving the implication of same key, window, and scope to the schema description.